analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -65,6 +67,8 @@ public void Main(bool boolParameter{additionalParameters})
         {methodBody}
     }}
 
+    public Exception CreateException() => new Exception();
```

Comment:
To be sure we support `System.Exception` and all it's inheritors. 
```suggestion
    public NotImplementedException CreateException() => new NotImplementedException();
```

Reply:
Cannot change this. Instead of checking the method invocations, we will check the conversion.

Reply:
Should be ok after using recursion.